release_notes/44-release-notes.adoc

```diff
@@ -0,0 +1,580 @@
++
+Checks have been added during integration creation to reduce the risk of issue creation failing after saving. A checkbox has also been introduced to give you the option to disable the priority setting.
+
+* Fixed an issue where an {product-title-short} policy with inform and enforce on deployment did not work when the deployment was scaled up. {product-title-short} has added `deployments/scale` in the resources and actions section of the validating webhook controller in the admission controller. 
++
+With this update, if you use the `oc scale` command to scale the deployment from 0 to a number, the admission controller blocks it if the deployment violates a policy.
+
+* Previously, a hanging network caused a random flake issue when you run the test scenario to open the user profile. With this update, this issue has now been fixed to improve the reliability of the test execution.
+
+* Previously, there were issues with compliance trigger scan calls aborting after 60 seconds due to synchronous execution in the backend. Your current scan progress could be lost if you navigated away and returned. This issue could affect you if you have numerous clusters and large data. The cause was the synchronous loading of data and the dependency on pending IDs, which are only determined during trigger scans.
++
+With this update, backend calls become asynchronous and an API parameter is introduced to retrieve the latest runs, enabling the user interface (UI) to recognize scans in progress. Improvements include loading running scans on page load, displaying progress and the ability to trigger additional scans during running scans.
+
+* Previously, editing a custom vulnerability report associated with a specific report scope in {product-title-short} 4.1 resulted in the loss of the report scope reference and would trigger `A report scope is required` message.
++
+The issue occurred when more than 10 collections were created and a vulnerability report edit was associated with 10 or more collections in {product-title-short} 4.2 and 4.3. The UI retrieved the first 10 collections from the API, possibly resulting in a missing reference to the report area. This behavior might persist in {product-title-short} 4.1 and 4.2. 
++
+This update ensures that the linked collection is retained, avoiding the error message.
+
+[id="image-versions_{context}"]
+== Image versions
+
+[%header,cols="1,1,2"]
+|===
+|Image |Description |Current version
+
+|Main
+|Includes Central, Sensor, Admission controller, and Compliance. Also includes `roxctl` for use in continuous integration (CI) systems.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}`
+
+|Scanner
+|Scans images and nodes.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:{rhacs-version}`
+
+|Scanner DB
+|Stores image scan results and vulnerability definitions.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}`
+
```

Comment:
Should also include the following:

```
|Scanner V4
|Scans images.
a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-v4-rhel8:{rhacs-version}`

|Scanner V4 DB
|Stores image scan results and vulnerability definitions for Scanner V4.
a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-v4-db-rhel8:{rhacs-version}`
```

Reply:
@msugakov @daynewlee can you help confirm these are the right names?

Reply:
based on Errata tool, that looks right:
https://errata.engineering.redhat.com/errata/container/128686